include/diplib/accumulators.h

```diff
@@ -621,7 +621,7 @@ class DIP_NO_EXPORT MomentAccumulator {
       }
 
       /// \brief Add a sample to the accumulator. `pos` must have `N` dimensions.
-      void Push( FloatArray pos, dfloat weight ) {
+      void Push( const FloatArray &pos, dfloat weight ) {
```

Comment:
Would you mind making this into `FloatArray const&`? I like keeping things consistent...